Mu2eUtilities/inc/ManagedList.hh

```diff
@@ -0,0 +1,67 @@
+//-----------------------------------------------------------------------------
+// P.Murat : a templated, light-weight, and "crippled" version of TClonesArray
+// assumptions:
+// - class T has an initialization method, say, T::Init, which properly reinitializes 
+//   an object allocated before
+// - an object of class T owns all its pointers (if any) and T::Init handles them correctly
+// - class T has a constructor T(int N), where N is the element index in the list
+// 
+// if those assumptions hold, a call to ManagedList::clear() 'empties' the list 
+// without reallocating the memory
```

Comment:
What is the use case for this class?  Why can't you use std::vector<unique_ptr<T>> instead?  That also manages memory, using standard tools.

Reply:
I will create an issue for removing this class and using std containers, which needs to be resolved after the PR is merged.